src/pages/ExportPrivateKey/EnterPassword.tsx

```diff
@@ -62,7 +71,53 @@ export function EnterPassword({
             <KeyIcon size={32} />
           </IconWrapper>
         </Stack>
-        <Stack sx={{ px: 2, mt: 3 }}>
+        {accountType === SecretType.Mnemonic && (
+          <Stack sx={{ px: 2, mt: 3 }}>
+            <Dropdown
+              SelectProps={{
+                defaultValue: PrivateKeyChain.C,
+                native: false,
+                displayEmpty: false,
+                renderValue: () => {
+                  switch (privateKeyChain) {
+                    case PrivateKeyChain.C:
+                      return <Typography>{t('C-Chain')}</Typography>;
+                    case PrivateKeyChain.XP:
+                      return <Typography>{t('X/P-Chain')}</Typography>;
+                  }
+                },
+                onChange: (e) => {
+                  const chain = e.target.value;
+                  if (chain && chain !== privateKeyChain) {
+                    setPrivateKeyChain(chain as PrivateKeyChain);
+                  }
+                },
+                // We need the @ts-ignore, because MUI's "nested props" (such as SelectProps)
+                // do not allow passing data-attributes.
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                'data-testid': 'private-key-chain-dropdown',
+              }}
+              label={t('Select chain')}
+            >
+              <DropdownItem
+                value={PrivateKeyChain.C}
+                selected={privateKeyChain === PrivateKeyChain.C}
+                data-testid="private-key-chain-dropdown-item"
```

Comment:
_nitpick:_ could be useful to differentiate the `data-testid`s between C and X/P